Docker/install.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+#########################################
+##        ENVIRONMENTAL CONFIG         ##
+#########################################
+
+# Configure user squeezeboxserver
+export DEBIAN_FRONTEND="noninteractive"
+useradd squeezeboxserver
+usermod -u 99 squeezeboxserver
+usermod -g 100 squeezeboxserver
```

Comment:
I would rather have that the UID and GID are configurable via environmental viariables $PUID and $PGID. This means this should not be a part of the install script, but be a part of the entrypoint script.

The reason for this: for instance Synology does not let the admin change UIDs or GIDs of its users, so to able to run this container under a seperate LMS user you need to be ale to change under which UID/GID the docker container runs.

Reply:
Hm, not sure i understand this. So synology would not let user creation and user permission setup inside a docker which is isolated from synology itself?

Reply:
If you create a LMS user in synology for running the docker container and be a owner of all the music data, you cannot control which UID or GID this user gets. If you then run the docker container with a other UID of GID (for instance 99 and 100 in your script), you cannot access the music files from within the container.

Reply:
Ok lets make it configurable so you can specify it as a variable and if they are not it will take default values.

Reply:
See https://github.com/jboesl/docker-logitechmediaserver/blob/master/sbs_entrypoint.sh for a solution.